### PR TITLE
Hotfixes

### PR DESCRIPTION
**Pull request by tompakarlsson**
I made two hotfixes to complete the story of the world.

### CODE REVIEW

**Comment by mayabrandi** [comment]
Hej

File: testrepo2.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('Good bye world...')
```
Comment (mayabrandi) [inline testrepo2.py:1]: Missing new line here!

File: testrepo3.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print('Where are you world?')
```
Comment (mayabrandi) [inline testrepo3.py:1]:
```suggestion
print('Where are you world?')

```

**Review by mayabrandi — COMMENTED** [review]
Good job. Had a few comments!

File: testrepo2.py (hunk 1 above)
Comment (tompakarlsson) [reply testrepo2.py]: Fixat

**Comment by tompakarlsson** [comment]
Please check my fixes @mayabrandi

**Comment by mayabrandi** [comment]
Perfect! Approving

**Review by mayabrandi — APPROVED** [review]
👍

**Pull request merged**